lib/matplotlib/axes/_axes.py

```diff
@@ -979,14 +979,14 @@ def hlines(self, y, xmin, xmax, colors='k', linestyles='solid',
 
         verts = [((thisxmin, thisy), (thisxmax, thisy))
                  for thisxmin, thisxmax, thisy in zip(xmin, xmax, y)]
-        coll = mcoll.LineCollection(verts, colors=colors,
+        lines = mcoll.LineCollection(verts, colors=colors,
                                     linestyles=linestyles, label=label)
```

Comment:
Shouldn't this line need to indent to keep things lined up (pep8 and all that...)